packages/wasm/crate/src/metadata.rs

```diff
@@ -70,6 +72,13 @@ pub fn customize_symbol_inner(metadata: Metadata) -> WasmResult<Metadata> {
             symbol: format!("VotedOn{proposal_id}"),
             ..metadata
         });
+    } else if let Some(captures) = lp_nft_re.captures(&metadata.base) {
+        let (state, id) = get_lp_info(&captures)?;
+
+        return Ok(Metadata {
+            symbol: format!("lpNft:{state}({id})"),
+            ..metadata
+        });
```

Comment:
Allows for LP position tokens to have symbols instead of displaying "Unknown Asset" in the UI.

Reply:
question: wouldn't it be more appropriate to use one of or some combination of the `display`, `description` or `name` fields?

it looks like ValueComponent doesn't use those at all for some reason.

Reply:
We've sort of reserved symbol as a trusted field given it's what ultimately will show to the users in the UI. The `display` field is for scaling the denom unit displayed. `description` and `name` are not shown and are considered untrusted.